packages/smooth_app/lib/knowledge_panel/knowledge_panels/knowledge_panel_world_map_card.dart

```diff
@@ -74,6 +65,16 @@ class KnowledgePanelWorldMapCard extends StatelessWidget {
               },
             )
           ],
+          children: <Widget>[
+            TileLayer(
+              urlTemplate: 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
+              subdomains: const <String>['a', 'b', 'c'],
```

Comment:
Actually now it's only `https://tile.openstreetmap.org/{z}/{x}/{y}.png`, no abc.

cf. https://wiki.openstreetmap.org/wiki/Raster_tile_providers

Reply:
Removed the subdomain 👍🏼